scenes/game/local_game.gd

```diff
@@ -4381,8 +4447,12 @@ func handle_strike_effect(card_id :int, effect, performing_player : Player):
 			performing_player.ignore_push_and_pull -= 1
 			if performing_player.ignore_push_and_pull == 0:
 				_append_log_full(Enums.LogType.LogType_Effect, performing_player, "no longer ignores pushes and pulls.")
+		"look_at_top_deck":
+			events += performing_player.reveal_topdeck()
 		"look_at_top_opponent_deck":
 			events += opposing_player.reveal_topdeck()
+		"look_at_opponent_hand":
```

Comment:
definitely should just be reveal_hand, opponent: true, and go add that down there.